man/login.defs.d/BTRFS_SUBVOLUME_HOME.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+   SPDX-FileCopyrightText: 2025, Hadi Chokr
+   SPDX-License-Identifier: BSD-3-Clause
+-->
+<varlistentry>
+  <term><option>BTRFS_SUBVOLUME_HOME</option> (boolean)</term>
```

Comment:
We should also indicate the default value. I don't think it's common in this project, but I think it's a good idea.